### DISCUSSION TITLE
Availability of sample code for GConv-LSTM, and GConv-GRU

### DISCUSSION THREAD

**Discussion by harirRzi** [pyg-team/pytorch_geometric#9274] [Q&A]
Hello. I want to implement node level prediction for each of GConv-LSTM, and GConv-GRU for my data that is in time series format. I have multiple graphs, each of them is for one timestep. The graphs are in different size and shapes. So I think I need to use dynamic temporal graph. I want to train GConv-LSTM, and GConv-GRU models based on some of those graphs and then predict the labels of each of the nodes for test part. Is there any code example that can help me regarding it?

Also is there any unsupervised or semi supervised for these models?

**Comment by rusty1s** [comment]
Is that related to https://github.com/benedekrozemberczki/pytorch_geometric_temporal?
